NanoAnalysis/python/nanoZZ4lAnalysis.py

```diff
@@ -27,17 +27,17 @@
                                     # "ULAPV", (used by LeptonSFHelper)
                                     # "pre_EE" (used by LeptonSFHelper, eleScaleResProducer, muonScaleResProducer, puWeightProducer, jetJERC)
                                     # "2022E", "2022F", "2022G" (used by jetJERC)
-                                    # "pre_BPix" (used by LeptonSFHelper, eleScaleResProducer, muonScaleResProducer, puWeightProducer, jetVMAP, jetJERC)
-NANOVERSION = getConf("NANOVERSION", 12)
-if not (LEPTON_SETUP == 2016 or LEPTON_SETUP == 2017 or LEPTON_SETUP == 2018 or LEPTON_SETUP == 2022 or LEPTON_SETUP == 2023) :
+#NANOVERSION = getConf("NANOVERSION", 12)
+NANOVERSION = getConf("NANOVERSION", 15)
+if not (LEPTON_SETUP == 2016 or LEPTON_SETUP == 2017 or LEPTON_SETUP == 2018 or LEPTON_SETUP == 2022 or LEPTON_SETUP == 2023 or LEPTON_SETUP == 2024 or LEPTON_SETUP == 2025) :
     print("Invalid LEPTON_SETUP", LEPTON_SETUP)
     exit(1)
 IsMC = getConf("IsMC", True)
 PD = getConf("PD", "")
 XSEC = getConf("XSEC", 1.)
 SYNCMODE = getConf("SYNCMODE", False) # fake smearing in Run2 correction modules, for synchronization purposes. No longer needed for Run3 modules.
 runMELA = getConf("runMELA", True)
-bestCandByMELA = getConf("bestCandByMELA", True) # requires also runMELA=True
+bestCandByMELA = getConf("bestCandByMELA", False) # requires also runMELA=True
```

Comment:
Again, changing defaults is tricky - it can change the behaviour for other users without them noticing it - I would rather put it in csv files if you don't want to forget to use  the `-a "bestCandByMELA=False;runMELA=False"` option in batch_Condor.py